arangod/IResearch/IResearchViewMeta.cpp

```diff
@@ -158,14 +159,30 @@ arangodb::iresearch::IResearchViewMeta::ConsolidationPolicy createConsolidationP
     }
   }
 
+  {
+    // optional double
+    static const std::string fieldName("minScore");
+
+    if (slice.hasKey(fieldName)) {
+      auto field = slice.get(fieldName);
+
+      if (!field.isNumber<size_t>()) {
```

Comment:
is it intentional that here you are checking for a `size_t` and use a `double` for the following `getNumber` call?

Reply:
well spotted, copy paste error